buildtest/cli/report.py

```diff
@@ -126,6 +128,11 @@ def __init__(
         self.color = color
         self.input_report = report_file
 
+        if format_detailed:
+            self.format = (
+                "name,id,user,state,returncode,runtime,outfile,errfile,buildspec"
```

Comment:
Create a class variable format_fields_detailed and declare this. Then assign self.format to this variable